mdata/aggmetrics.go

```diff
@@ -66,8 +66,10 @@ func (ms *AggMetrics) GC() {
 		ms.RUnlock()
 		for _, org := range orgs {
 			orgActiveMetrics := promActiveMetrics.WithLabelValues(strconv.Itoa(int(org)))
-			keys := make([]schema.Key, 0, len(ms.Metrics[org]))
+			// need to acquire lock here, otherwise can run into panic because
+			// GetOrCreate might be writing to ms.Metrics
```

Comment:
let's not document minutiae such as GetOrCreate being the only thing that happens to write to the map.
this is an implementation detail right now that can change in the future.
this is too brittle.
the only relevant context here in this scope is that any other code could write to the map.
and actually this seems so obvious we don't need a comment at all.
check out the commit i just added.